syft/frameworks/torch/hook/hook.py

```diff
@@ -147,6 +147,8 @@ def __init__(
         if dependency_check.crypten_available:
             self.to_auto_overload[crypten.mpc.MPCTensor] = ["get_plain_text"]
             self._hook_syft_placeholder_methods(crypten.mpc.MPCTensor, PlaceHolder)
+            self.to_auto_overload[crypten.nn.Module] = ["encrypt", "__call__", "train", "size"]
```

Comment:
This list could perhaps be defined in syft/frameworks/crypten/hook/hook.py instead, where you already define some methods to be hooked?

Reply:
Done